core/ir/x86/decode_table.c

```diff
@@ -75,7 +75,7 @@
 const instr_info_t * const op_instr[] =
 {
     /* OP_INVALID */   NULL,
-    /* OP_UNDECODED */ NULL,
+    /* OP_UNDECODED */ (instr_info_t[]){{1, 0xffffff, 0, "<undecoded>", 0, 0, 0, 0, 0, 0, 0, 0}},
```

Comment:
The 1 should be OP_UNDECODED right?

Reply:
Yes, but OP_UNDECODED it's defined below and I didn't want to overturn this file too much just for that.
I can move `op_instr[]` down below though.

Reply:
The enum value is defined in opnd_api.h which must be included before this right?

Reply:
Yes, it's in opcode_api.h, I'm not sure why I was getting an error from clangd when using the enum name.
s/1/OP_UNDECODED
Done.

Reply:
No more relevant. https://github.com/DynamoRIO/dynamorio/pull/6663#discussion_r1586981783